lib/EarlyFunctionAnalysis/Outliner.cpp

```diff
@@ -393,12 +395,7 @@ Outliner::outlineFunctionInternal(CallHandler *TheCallHandler,
 
   if (not HasEntryBlock) {
     auto *EntryBlock = BasicBlock::Create(Root->getContext(), "", Root);
-    Function *AbortFunction = notNull(Root->getParent()->getFunction("_abort"));
-    IRBuilder<> Builder(EntryBlock);
-    emitCall(Builder,
-             AbortFunction,
-             "This starts at non-executable address",
-             DebugLoc());
+    abortWithMessage(EntryBlock, "This starts at non-executable address");
```

Comment:
Unifying is nice but `abortWithMessage` does not convey that we're *emitting* an abort, and not aborting revng itself. `emitAbort`?

Reply:
Done.